test/e2e/common.go

```diff
@@ -793,3 +799,211 @@ func GetCAPM3StableReleaseOfMinor(ctx context.Context, minorRelease string) (str
 	releaseMarker := fmt.Sprintf(releaseMarkerPrefix, minorRelease)
 	return clusterctl.ResolveRelease(ctx, releaseMarker)
 }
+
+// GitCheckout checkouts the repo that is at path repoPath to branch checkoutBranch.
+func GitCheckout(repoPath string, checkoutBranch string) error {
```

Comment:
There isn't a lib for this?

Reply:
not using it anywhere, added it for testing. Now it's removed.